com.github.mtkennerly.ludusavi.metainfo.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+	<id>com.github.mtkennerly.ludusavi</id>
+	<metadata_license>CC0-1.0</metadata_license>
+	<project_license>MIT</project_license>
+	<name>ludusavi</name>
+	<developer_name>mtkennerly</developer_name>
+	<url type="homepage">https://rambox.app/#ce</url>
```

Comment:
are you sure?

Reply:
oh, this slipped trough when converting an old metainfo